ref/Arithmetic_Operators.md

```diff
@@ -318,10 +324,13 @@ The following operators generate pseudo random numbers.
 #### Uniformly distributed random real number between 0 and `‹number›`: `random(‹number›)`
 
     > random(10)
-    < 7.089078226464412
-    ~ \d\.\d{3,}
+    < 7.089
+    ~ \d\.\d+
```

Comment:
Only if you care to cater for the case where the random number is exactly equal to zero, at least after rounding. If we were to do that here, we should do so for all those number patterns, since this situation is even more likely for the nullary `random()` call.